edk2toollib/windows/policy/firmware_policy_test.py

```diff
@@ -0,0 +1,202 @@
+        73 39 5F 3E AB 5C 60 EC DF 03 10 A8 D3 09 E9 F4 F6 96 85 B6 7F 51 88 66
+        47 19 8D A2 B0 12 3D 81 2A 68 05 77 BB 91 4C 62 7B B6 C1 07 C7 BA 7A 87
+        34 03 0E 4B 62 7A 99 E9 CA FC CE 4A 37 C9 2D A4 57 7C 1C FE 3D DC B8 0F
+        5A FA D6 C4 B3 02 85 02 3A EA B3 D9 6E E4 69 21 37 DE 81 D1 F6 75 19 05
+        67 D3 93 57 5E 29 1B 39 C8 EE 2D E1 CD E4 45 73 5B D0 D2 CE 7A AB 16 19
+        82 46 58 D0 5E 9D 81 B3 67 AF 6C 35 F2 BC E5 3F 24 E2 35 A2 0A 75 06 F6
+        18 56 99 D4 78 2C D1 05 1B EB D0 88 01 9D AA 10 F1 05 DF BA 7E 2C 63 B7
+        06 9B 23 21 C4 F9 78 6C E2 58 17 06 36 2B 91 12 03 CC A4 D9 F2 2D BA F9
+        94 9D 40 ED 18 45 F1 CE 8A 5C 6B 3E AB 03 D3 70 18 2A 0A 6A E0 5F 47 D1
+        D5 63 0A 32 F2 AF D7 36 1F 2A 70 5A E5 42 59 08 71 4B 57 BA 7E 83 81 F0
+        21 3C F4 1C C1 C5 B9 90 93 0E 88 45 93 86 E9 B1 20 99 BE 98 CB C5 95 A4
+        5D 62 D6 A0 63 08 20 BD 75 10 77 7D 3D F3 45 B9 9F 97 9F CB 57 80 6F 33
+        A9 04 CF 77 A4 62 1C 59 7E 31 82 02 15 30 82 02 11 02 01 01 30 81 9C 30
+        81 84 31 0B 30 09 06 03 55 04 06 13 02 55 53 31 13 30 11 06 03 55 04 08
+        13 0A 57 61 73 68 69 6E 67 74 6F 6E 31 10 30 0E 06 03 55 04 07 13 07 52
+        65 64 6D 6F 6E 64 31 1E 30 1C 06 03 55 04 0A 13 15 4D 69 63 72 6F 73 6F
+        66 74 20 43 6F 72 70 6F 72 61 74 69 6F 6E 31 2E 30 2C 06 03 55 04 03 13
+        25 4D 69 63 72 6F 73 6F 66 74 20 57 69 6E 64 6F 77 73 20 50 72 6F 64 75
+        63 74 69 6F 6E 20 50 43 41 20 32 30 31 31 02 13 33 00 00 00 24 18 FC 0B
+        68 9E 73 99 D0 00 00 00 00 00 24 30 0D 06 09 60 86 48 01 65 03 04 02 01
+        05 00 A0 4B 30 18 06 09 2A 86 48 86 F7 0D 01 09 03 31 0B 06 09 2B 06 01
+        04 01 82 37 4F 01 30 2F 06 09 2A 86 48 86 F7 0D 01 09 04 31 22 04 20 C5
+        90 2F B4 18 90 76 D6 F4 5A EE 0B 7F 26 5F 40 5C FE 09 51 5A BE FA C5 4A
+        B3 52 9B 3D 82 BB 9F 30 0D 06 09 2A 86 48 86 F7 0D 01 01 01 05 00 04 82
+        01 00 72 4B 8C DA 85 D2 1E C0 71 68 99 77 79 2E 51 78 52 4B F6 9F 11 31
+        E7 D5 F2 00 10 CD 94 B9 D3 C7 67 B4 EA F4 2B 62 04 02 93 55 79 97 A2 D0
+        EA 57 F0 81 56 42 1E 10 5E 18 79 8F 59 AB DB 86 EC A3 74 37 48 FE 91 3B
+        48 7C 94 CF FE D3 D6 44 0B 94 D2 0C 6F 12 20 9C 92 77 EB 3A 8C 84 BB 81
+        93 AD 5A FF F8 2B 27 C8 9C 61 85 6C 1A D5 2B B6 1C D5 7A EA 86 5C 08 A0
+        CB 51 21 49 4B F2 04 36 5F 38 48 80 D0 CD FC E5 4D 6A 19 0A 17 86 8D E7
+        4B 32 09 62 15 95 36 61 1C 07 12 B8 D7 64 FA CE 74 84 42 20 18 77 A5 F4
+        4E ED 42 5C 64 BD 57 99 E7 C3 7E 2D 0C 8F FE 38 7B 68 0C 78 DF 3D E9 30
+        10 EA D4 1E 24 E1 60 F1 89 92 59 B4 8C F5 F6 8C F9 CE 8C EB 99 D6 35 0A
+        03 0E FB 88 4A 01 54 8C 20 84 18 F5 49 6B 8B 96 D3 55 45 0C 99 79 23 28
+        51 18 C2 4B 96 02 74 79 86 44 FB B5 98 59 69 E2 80 DE""")
+
+    def test_basic_decode(self):
+        inp = io.BytesIO(FirmwarePolicyTest.TEST_DEVICEID_C2E28)
```

Comment:
maybe you should specify what C2E28 is and then compare policyManuf.<fields here> with their actual values.

Reply:
Upcoming commit will do this and more

Reply:
Addressed in commit 2a966083e3b43c04a5f6c6ab338108bcdfe9244c